json_spec/README.md

```diff
@@ -2033,6 +2063,57 @@ Provides information about software that was discovered or exercised during the
 ```
 
 
+#### SourceLocation
+
+##### Description
+
+Provides information about source location for the purpose of debugging or tracing program execution.
```

Comment:
i still think this isnt explicit enough to say that the reference for these lines/files is the diagnostic package itself

Reply:
same with the individual fields

Reply:
updated